assets/javascripts/discourse/components/events-calendar-subscription.js.es6

```diff
@@ -5,22 +5,43 @@ export default DropdownSelectBoxComponent.extend({
   classNames: ["events-calendar-subscription"],
   rowComponent: "events-calendar-subscription-row",
 
-  @computed
-  content() {
+  @computed('authSuffix')
+  content(authSuffix) {
     const baseUrl = window.location.host + window.location.pathname;
     const timeZone = moment.tz.guess();
     return [
       {
-        id: `webcal://${baseUrl}.ics?time_zone=${timeZone}`,
+        id: `webcal://${baseUrl}.ics?time_zone=${timeZone}${authSuffix}`,
```

Comment:
Add authSuffix only if the site is private

Reply:
Should work now